README.md

```diff
@@ -175,31 +175,31 @@ Install-Module InvokeBuild -Scope CurrentUser
 Install-Module platyPS -Scope CurrentUser
 ```
 
-### 4. Delete `NuGet.Config`
+### 4. Adjust `Nuget.Config` if necessary
 
-Our NuGet configuration points to a private feed necessary for secure builds,
-and it must be committed to the repo as it is.
-The easiest way to build without access to that private feed is to delete the file:
+Our NuGet configuration uses a secure feed with whitelisted third party dependency packages. If your contribution requires any changes to the included NuGet packages, you must disable this secure feed.
+
+First, run this command to prevent accidentally commiting changes to this file
 
 ```powershell
-Remove-Item NuGet.Config
+git update-index --skip-worktree nuget.config
 ```
 
-Please be careful not to commit this change in a PR.
+Then, either delete the file or remove the `packagesources` section to use nuget.org again. Your PR _will_ fail automated build checks and you _must_ inform us at the top of your PR so the appropriate packages can be added if approved.
 
-Now you're ready to build the code.
-You can do so in one of two ways:
+## Build PowerShell Editor Services
+Now you're ready to build the code. You can do so in one of two ways:
 
-### Building the code from PowerShell
+### PowerShell
 
 ```powershell
 PS C:\path\to\PowerShellEditorServices> Invoke-Build Build
```

Comment:
```suggestion
PS C:\src\PowerShellEditorServices> Invoke-Build
```

Might as well use the default task.

Reply:
The default task runs tests as well, wasn't sure if that was preferred.